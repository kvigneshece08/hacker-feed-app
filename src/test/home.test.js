import Home from '../home/home';
import { shallow } from 'enzyme';
import React from 'react';
import Adapter from '../jest-config';

it("should render initial layout", () => {
    const component = shallow(<Home />);
    expect(component.getElements()).toMatchSnapshot();
});