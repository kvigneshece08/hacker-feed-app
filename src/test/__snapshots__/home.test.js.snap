// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render initial layout 1`] = `
Array [
  <div>
    <React.Fragment>
      <h1>
        Hackers feed
      </h1>
      <h3>
        Loading...
      </h3>
    </React.Fragment>
  </div>,
]
`;
